src/styles/theme/oc-button.scss

```diff
@@ -39,15 +64,85 @@
 
   &.uk-button-large {
     min-height: $global-control-large-height;
+    font-size: 1.5rem;
   }
 
   &.uk-button-primary,
   &.uk-button-secondary,
   &.uk-button-danger {
+    border: none;
+
     .oc-icon > svg {
       fill: $global-inverse-color;
     }
   }
+
+  &.uk-button-default {
+    border: 1px solid $action-primary;
+    background-color: transparent;
+    color: $action-primary;
+
+    .oc-icon > svg {
+      fill: $action-primary;
+    }
+
+    &:hover,
+    &:focus {
+      border-color: $action-primary-hover;
+      color: $action-primary-hover;
+
+      .oc-icon > svg {
+        fill: $action-primary-hover;
+      }
+    }
+  }
+
+  &.uk-button-primary {
+    border: none;
+    background-color: $action-primary;
+    box-shadow: 0 4px 6px rgba($shadow-action-primary, 0.11), 0 1px 3px rgba($shadow-global, 0.08);
+    color: white;
+
+    &:hover,
+    &:focus {
+      background-color: $action-primary-hover;
+    }
+  }
+
+  .uk-button-group & {
+    border-radius: 0;
+    // Prevent negative margin which is coming from UIkit
+    margin-left: 0 !important;
+
+    &:first-of-type {
+      border-radius: 3px 0 0 3px;
+    }
+  
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)